lib/manageiq/api/common/headers.rb

```diff
@@ -11,6 +12,11 @@ def self.current
           Thread.current[:attr_current_headers]
         end
 
+        def self.forwardable_headers
+          raise "Current Headers has not been set" unless Thread.current[:attr_current_headers]
```

Comment:
`"current header have not been set"`

Also wondering if this should raise a specific error other than RuntimeError, or perhaps not raise an exception at all, and instead just return `{}`

Reply:
We'll need this in order to return a useful error document